mpbb-install-port

```diff
@@ -108,6 +112,18 @@ install-port() {
         fi
         return 1
     fi
+
+    if [[ "${option_test}" -eq 1 ]]; then
+        if "${option_prefix}/bin/port" "-d" test "$@"; then
+            echo "Testing '$port' ... [OK]" >> "$log_subports_progress"
```

Comment:
With the current base release, this will fail for any port that doesn't set `test.run yes`.

Reply:
@jmroot may I assume that the next release which will happen relatively soon changes that behaviour?